src/lib/label/DY.Lib.Label.DynamicLabelEvent.fst

```diff
@@ -0,0 +1,59 @@
+module DY.Lib.Label.DynamicLabelEvent
+
+open Comparse
+open DY.Core
+open DY.Lib.Event.Typed
+open DY.Lib.Comparse.Glue
+open DY.Lib.Comparse.Parsers
+
+[@@ with_bytes bytes]
+type reveal_event_format = {
+  [@@@ with_parser #bytes ps_string]
+  to:string;
```

Comment:
Nits:
- This should probably be `principal`, even if the two types are equivalent
- Something like `revealed_to` might be a more explicit name than `to` --- likewise, maybe `point` would be clearer as something like the `nonce_at` used later, though that's still not fully descriptive.